src/hooks/use-server-component-page-params.tsx

```diff
@@ -6,7 +6,13 @@ import getPageQueryParamsValues from './use-page-query-params/helpers/get-page-q
 
 export function useServerComponentQueryParams<P extends PageQueryParams>(
```

Comment:
```suggestion
export function useServerQueryParams<P extends PageQueryParams>(
```

Thoughts on this? Since we're using this in the server side for route handlers too

Reply:
Since we're using Zod for query param validation in route handlers, this is unnecessary